src/staging/staging_main.py

```diff
@@ -144,7 +139,7 @@ def run_staging(  # noqa: C901
 
         # Write both snapshots to feather file at given path
         if is_network:
-            feather_fname = f"{snapshot_name}_corrected.feather"
+            feather_fname = f"{snapshot_name}.feather"
```

Comment:
Why had this previously been called _corrected?  I can't see any reason why it should be called that - just curious :)

Reply:
we were updating the feather file, but needed to do this as we didn't want to break it for those still using the old version.